Classes/Projection/ProjectionManager.php

```diff
@@ -93,22 +88,7 @@ public function getProjection(string $projectionIdentifier): Projection
     {
         $fullProjectionIdentifier = $this->normalizeProjectionIdentifier($projectionIdentifier);
         $projectorClassName = $this->projections[$fullProjectionIdentifier];
-        $eventClassNames = $this->eventListenerLocator->getEventClassNamesByListenerClassName($projectorClassName);
-        return new Projection($fullProjectionIdentifier, $projectorClassName, $eventClassNames);
-    }
-
-    /**
-     * Tells if the specified projection is currently empty
-     *
-     * @param string $projectionIdentifier
-     * @return bool
-     */
-    public function isProjectionEmpty(string $projectionIdentifier): bool
```

Comment:
Is this method not needed any more?

Reply:
No, it was just a left-over.
We might need something similar that allows listeners to specify some additional filters (See #141) but currently that part was not invoked anyways and IMO it should not matter whether a projection is "empty" – what does that mean anyways? :)

Reply:
Well, empty means "no data/nothing projected" but fine with this. Was just concerned about breakiness

Reply:
Right – but IMO that should not be determined by the state of the projection but by the "highest applied sequence number". So the projector itself doesn't "know" about it necessarily